api/tests/routes/backoffice_v3/accounts_test.py

```diff
@@ -947,26 +947,18 @@ def test_update_email_triggers_history_token_and_mail(self, authenticated_client
         # check that email has been changed immediately after admin request
         db.session.refresh(user)
         assert user.email == "updated@example.com"
-        assert not user.isEmailValidated
+        assert user.isEmailValidated
 
-        # check that a line has been added in email history
+        # check that two rows has been added in email history (update and validate)
         email_history: list[users_models.UserEmailHistory] = users_models.UserEmailHistory.query.filter(
             users_models.UserEmailHistory.userId == user.id
         ).all()
         assert len(email_history) == 1
-        assert email_history[0].eventType == users_models.EmailHistoryEventTypeEnum.ADMIN_UPDATE_REQUEST
-        assert email_history[0].oldEmail == "gg@example.net"
-        assert email_history[0].newEmail == "updated@example.com"
 
-        # check that a new token has been generated
-        token: users_models.Token = users_models.Token.query.filter(users_models.Token.userId == user.id).one()
-        assert token.type == users_models.TokenType.EMAIL_VALIDATION
-
-        # check that email is sent
-        assert len(mails_testing.outbox) == 1
-        assert mails_testing.outbox[0].sent_data["To"] == "updated@example.com"
-        assert mails_testing.outbox[0].sent_data["template"] == TransactionalEmail.EMAIL_CONFIRMATION.value.__dict__
-        assert token.value in mails_testing.outbox[0].sent_data["params"]["CONFIRMATION_LINK"]
+        expected_event_types = {users_models.EmailHistoryEventTypeEnum.ADMIN_UPDATE}
+        assert {event.eventType for event in email_history} == expected_event_types
+        assert {event.oldEmail for event in email_history} == {"gg@example.net"}
+        assert {event.newEmail for event in email_history} == {"updated@example.com"}
```

Comment:
Comparer des `set` d'un élément nuit vraiment à la lisibilité
```suggestion
        assert email_history[0].eventType == users_models.EmailHistoryEventTypeEnum.ADMIN_UPDATE
        assert email_history[0].oldEmail == "gg@example.net"
        assert email_history[0].newEmail == "updated@example.com"
```